src/index.html

```diff
@@ -15,8 +15,23 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html"><img  src="images/logo.png" alt="Moyo" > </img></a>
+      <nav>
+        <ul class="nav_list">
+          <li class="nav_item"><a  class="is-active nav_link">Apple</a></li>
+          <li class="nav_item"><a  class="nav_link">Samsung</a></li>
+          <li class="nav_item"><a  class="nav_link">Smartphones</a></li>
+          <li class="nav_item"><a data-qa="hover" class="nav_link">Laptops & Computers</a></li>
+          <li class="nav_item"><a  class="nav_link">Gadgets</a></li>
+          <li class="nav_item"><a  class="nav_link">Tablets</a></li>
+          <li class="nav_item"><a  class="nav_link">Photo</a></li>
+          <li class="nav_item"><a  class="nav_link">Video</a></li>
+        </ul>
+      </nav>
```

Comment:
Remember about correct indentation between parent and child elements. Each level of nesting, including text, contained inside the element, requires 2-space offset. Also blank line shouldn't be between parent and child elements.